plugins/grpc/client_sendmsg_interceptor.go

```diff
@@ -0,0 +1,73 @@
+// Licensed to Apache Software Foundation (ASF) under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Apache Software Foundation (ASF) licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package grpc
+
+import (
+	"github.com/apache/skywalking-go/plugins/core/operator"
+	"github.com/apache/skywalking-go/plugins/core/tracing"
+	"google.golang.org/grpc/peer"
+)
+
+type ClientSendMsgInterceptor struct {
+}
+
+func (h *ClientSendMsgInterceptor) BeforeInvoke(invocation operator.Invocation) error {
+
+	cs := invocation.CallerInstance().(*nativeclientStream)
+	ctx := cs.Context()
+	method := cs.callHdr.Method
+	peerKey, ok := peer.FromContext(ctx)
+	remoteAddr := ""
+	if ok {
+		remoteAddr = peerKey.Addr.String()
+	}
+
+	if remoteAddr == "127.0.0.1:11800" {
+		return nil
+	}
+
+	s, err := tracing.CreateLocalSpan(formatOperationName(method, "/client/Request/sendMsg"),
```

Comment:
I think all the name should have same case rule. You can choose to capitalize the first letter of all words or not.